tests/e2e/e2e_upgrade_test.go

```diff
@@ -49,7 +49,7 @@ var _ = OSMDescribe("Upgrade from latest",
 			// release branches, it should specify the most recent patch of the
 			// previous minor release. e.g. on the release-v1.0 branch, this
 			// should be "0.11".
-			i.Version = ">0.0.0-0"
+			i.Version = ">1.1.1"
```

Comment:
We don't want this to match v1.2+ which is why we omit the `>`. Otherwise this "upgrade" could go backwards which we don't want.
```suggestion
			i.Version = "1.1"
```

Reply:
`1.1` will match the latest patch so it will resolve to `1.1.1`.